source/dagger/core/graphics/window.cpp

```diff
@@ -179,7 +180,14 @@ void WindowSystem::SpinUp()
 	}
 
 	glfwMakeContextCurrent(window);
-    glfwSwapInterval(0);
+	if (m_Config.vsync)
+	{
+    	glfwSwapInterval(1);
```

Comment:
Might just be github, but indentation seems to be a bit bad here

Reply:
For some reason only that line mixes spaces and tabs